### ISSUE TITLE
OpenKM DMS connector integration - Phase 1

### ISSUE THREAD

**Issue by calexh-sar** [sartography/spiff-arena#580]
Connect to OpenKM API to allow the following actions:

**Phase 1**
1. Upload a file — all files that we currently write to MySQL should be written to the DMS
2. GET file(s) — using the returned GUID, fetch the file itself 
3. Update metadata

**API documentation**
https://docs.openkm.com/kcenter/view/okm-8.1/restful.html

OpenKM
https://test-docs.logos.co/openkm/kcenter/

https://www.openkm.com/wiki/index.php/RESTful_Guide - deprecated

**Comment by calexh-sar** [comment]
Waiting on more info from Harmeet regarding a demo account.

**Comment by mrtrosen** [comment]
The latest documentation for the API has moved to their docs website:  https://docs.openkm.com/apidoc/
The link in a previous git comment is deprecated.

**Comment by calexh-sar** [comment]
Move this to New Issue since it needs to be re-estimated based on recently discovered API docs.

**Comment by madhurrya** [comment]
@calexh-sar please check this one
